website/data/language-nav-data.json

```diff
@@ -337,6 +337,10 @@
             "title": "<code>startswith</code>",
             "href": "/language/functions/startswith"
           },
+          {
+            "title": "<code>strcontains</code>",
+            "href": "/language/functions/strcontains"
```

Comment:
@kmoe  Do you know why the CI is not picking this up?

Reply:
Hello, Docs team here! 👋  Our new content checker bot isn't set up to handle this navigational file format yet, so we've added a task to fix that in the future. In the meantime, this file is linking properly, so @Cliftonz you are good to merge from our side of things!

Reply:
Per @alisdair, I think this is because we are missing a secondary entry in this same file. We recently added a similar function called `plantimestamp` - you can see four total references to this function in the file (https://github.com/hashicorp/terraform/blob/main/website/data/language-nav-data.json). The entry should be something like:

`	  { "title": "strcontains", "path": "functions/strcontains", "hidden": true},`

I've created a pull request which I believe will fix the issue. https://github.com/hashicorp/terraform/pull/33101

Reply:
Thank you so much for following up on this @crw; I've approved your follow up PR!